weave

```diff
@@ -1759,11 +1828,13 @@ launch_router() {
         --ipalloc-range "$IPRANGE" \
         --dns-effective-listen-address $DOCKER_BRIDGE_IP \
         $DNS_ROUTER_OPTS $NO_DNS_OPT \
+        $AWSVPC_ARGS \
         --http-addr $HTTP_ADDR \
         "$@"
     setup_router_iface_$BRIDGE_TYPE
     wait_for_status $CONTAINER_NAME http_call $HTTP_ADDR
     [ -n "$ALREADY_RUNNING_CONTAINER" ] || populate_router
+    [ -z "$AWSVPC" ] || expose
```

Comment:
It's already done in https://github.com/weaveworks/weave/pull/2091/files#diff-ed6579eaedd0e67b3ed51c7ee7071044R2085